### PR TITLE
Include hyperlinks in command summary

### LINKED ISSUE #270: [PE-D][Tester C] UG Documentation: Include links from command summary to actual command in feature section
More of a suggestion to improve user experience of using UG. Could include link in command summary for each action to bring the user to the command in the feature section to see more info.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: shaunlxw/ped#11

### PR DESCRIPTION
**Pull request by ruijietay**
Resolves #270.

### CODE REVIEW

**Review by dillontkh — COMMENTED** [review]
Since the feature sections are already markdown style headers why not use markdown style linking instead of HTML anchor tags so that it can be checked by the GitHub action?

**Review by jjchee77 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| [**FitAdd**](#fitadd)       | `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `<br> e.g., `fitadd 1 n/burpees s/3 r/5 b/30`       |
+| [**FitDelete**](#fitdelete) | `fitdelete INDEX [n/EXERCISE_NAME] [/all]`<br> e.g., `fitdelete 1 n/burpees`                                                                                                                        |
+| [**List**](#list)           | `list`                                                                                                                                                                                              |
+| [**Help**](#help)           | `help`                                                                                                                                                                                              |
+| [**Note**](#note)           | `note INDEX [NOTE]` <br> e.g. <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
```
Comment (bgopi23) [inline docs/UserGuide.md:413]:
```suggestion
| [**Note**](#note)           | `note INDEX [NOTE]` <br> e.g., <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| [**Weight**](#weight)       | `WEIGHT INDEX [WEIGHT]` <br> e.g. <br> `weight 3 70`                                                                                                                                                |
```
Comment (bgopi23) [inline docs/UserGuide.md:414]:
```suggestion
| [**Weight**](#weight)       | `WEIGHT INDEX [WEIGHT]` <br> e.g., <br> `weight 3 70`                                                                                                                                                |
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
-| **List**      | `list`                                                                                                                                                                                              |
-| **Help**      | `help`                                                                                                                                                                                              |
-| **Note**      | `note INDEX [NOTE]` <br> e.g. <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
-| **Weight**    | `WEIGHT INDEX [WEIGHT]` <br> e.g. <br> `weight 3 70`                                                                                                                                                |
+| Action                      | Format, Examples                                                                                                                                                                                    |
```
Comment (jjchee77) [inline docs/UserGuide.md:401]: Should we change "Action" to "Command" instead

File: docs/UserGuide.md [hunk 4]
```diff
@@ -245,6 +252,7 @@ Examples:
 * `fitdelete 2 /all` - Deletes all exercise(s) from the 2nd client.
 <hr>
 
+<a id="weight"></a>
 ### Adding a weight value to a client : `weight`
```
Comment (jjchee77) [inline docs/UserGuide.md:256]: Should we change this to "Adding or removing a weight"

File: docs/UserGuide.md [hunk 5]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| [**FitAdd**](#fitadd)       | `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `<br> e.g., `fitadd 1 n/burpees s/3 r/5 b/30`       |
+| [**FitDelete**](#fitdelete) | `fitdelete INDEX [n/EXERCISE_NAME] [/all]`<br> e.g., `fitdelete 1 n/burpees`                                                                                                                        |
+| [**List**](#list)           | `list`                                                                                                                                                                                              |
+| [**Help**](#help)           | `help`                                                                                                                                                                                              |
+| [**Note**](#note)           | `note INDEX [NOTE]` <br> e.g., <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
```
Comment (bgopi23) [inline docs/UserGuide.md:413]:
```suggestion
| [**Note**](#note)           | `note INDEX [NOTE]` <br> e.g. <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| [**Edit**](#edit)           | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                                               |
+| [**Exit**](#exit)           | `exit`                                                                                                                                                                                              |
+| [**Find**](#find)           | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                          |
+| [**FitAdd**](#fitadd)       | `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `<br> e.g., `fitadd 1 n/burpees s/3 r/5 b/30`       |
+| [**FitDelete**](#fitdelete) | `fitdelete INDEX [n/EXERCISE_NAME] [/all]`<br> e.g., `fitdelete 1 n/burpees`                                                                                                                        |
```
Comment (bgopi23) [inline docs/UserGuide.md:410]:
```suggestion
| [**FitDelete**](#fitdelete) | `fitdelete INDEX [n/EXERCISE_NAME] [/all]`<br> e.g. `fitdelete 1 n/burpees`                                                                                                                        |
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| [**Delete**](#delete)       | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                 |
+| [**Edit**](#edit)           | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                                               |
+| [**Exit**](#exit)           | `exit`                                                                                                                                                                                              |
+| [**Find**](#find)           | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                          |
+| [**FitAdd**](#fitadd)       | `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `<br> e.g., `fitadd 1 n/burpees s/3 r/5 b/30`       |
```
Comment (bgopi23) [inline docs/UserGuide.md:409]:
```suggestion
| [**FitAdd**](#fitadd)       | `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `<br> e.g. `fitadd 1 n/burpees s/3 r/5 b/30`       |
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| [**Clear**](#clear)         | `clear`                                                                                                                                                                                             |
+| [**Delete**](#delete)       | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                 |
+| [**Edit**](#edit)           | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                                               |
+| [**Exit**](#exit)           | `exit`                                                                                                                                                                                              |
+| [**Find**](#find)           | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                          |
```
Comment (bgopi23) [inline docs/UserGuide.md:408]:
```suggestion
| [**Find**](#find)           | `find KEYWORD [MORE_KEYWORDS]`<br> e.g. `find James Jake`                                                                                                                                          |
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
+| Action                      | Format, Examples                                                                                                                                                                                    |
+|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Add**](#add)             | `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 nt/likes pizzas t/friend t/colleague` |
+| [**Clear**](#clear)         | `clear`                                                                                                                                                                                             |
+| [**Delete**](#delete)       | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                 |
```
Comment (bgopi23) [inline docs/UserGuide.md:405]:
```suggestion
| [**Delete**](#delete)       | `delete INDEX`<br> e.g. `delete 3`                                                                                                                                                                 |
```

File: docs/UserGuide.md [hunk 10]
```diff
@@ -386,17 +398,18 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
...
-| **Note**      | `note INDEX [NOTE]` <br> e.g. <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
-| **Weight**    | `WEIGHT INDEX [WEIGHT]` <br> e.g. <br> `weight 3 70`                                                                                                                                                |
+| Action                      | Format, Examples                                                                                                                                                                                    |
+|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Add**](#add)             | `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 nt/likes pizzas t/friend t/colleague` |
```
Comment (bgopi23) [inline docs/UserGuide.md:403]:
```suggestion
| [**Add**](#add)             | `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​` <br> e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 nt/likes pizzas t/friend t/colleague` |
```

**Review by bgopi23 — APPROVED** [review]
LGTM!

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM!

**Pull request merged**